### PR TITLE
print operations in verbose mode of cli get node command

### PR DESCRIPTION
**Pull request by idob-cloudify**

### CODE REVIEW

File: cloudify_cli/commands/nodes.py [hunk 1]
```diff
@@ -87,6 +90,21 @@ def get(node_id, deployment_id, logger, client, tenant_name):
         logger.info('\t{0}: {1}'.format(property_name, property_value))
     logger.info('')
 
+    if get_global_verbosity() != NO_VERBOSE:
+        operations = []
+        for op in utils.decode_dict(
```
Comment (mcouthon) [inline cloudify_cli/commands/nodes.py]: If working with `iteritems` it's often clearer to do:
```
for op_name, op in utils.decode_dict(node.operations).iteritems():
    op['name'] = op_name
    operations += op
```

File: cloudify_cli/commands/nodes.py [hunk 2]
```diff
@@ -87,6 +90,21 @@ def get(node_id, deployment_id, logger, client, tenant_name):
...
+                node.operations).iteritems():
+            # op is a tuple (operation_name, dict_of_attributes)
+            # we want to add the name to the dict
+            # and build a new array in order to print it in a table
+            tempdict = op[1].copy()
+            tempdict.update({'name': op[0]})
+            operations = operations + [tempdict]
+            # logger.info('\t{0}'.format(op))
```
Comment (mcouthon) [inline cloudify_cli/commands/nodes.py]: Probably a leftover

**Comment by mcouthon** [comment]
Do note, also, that when one of the fields is large (for example, the inputs field of some operation), then the table looks pretty broken.
This is due to how we create tables, but still.

File: cloudify_cli/commands/nodes.py (hunk 1 above)
Comment (idob-cloudify) [reply cloudify_cli/commands/nodes.py]: fixed, just small update:
  operations += [op]

File: cloudify_cli/commands/nodes.py [hunk 3]
```diff
@@ -87,6 +90,21 @@ def get(node_id, deployment_id, logger, client, tenant_name):
         logger.info('\t{0}: {1}'.format(property_name, property_value))
     logger.info('')
 
+    if get_global_verbosity() != NO_VERBOSE:
+        operations = []
+        for op in utils.decode_dict(
+                node.operations).iteritems():
```
[the remark this answers is not in the document]
Comment (idob-cloudify) [reply cloudify_cli/commands/nodes.py]: fixed

File: cloudify_cli/commands/nodes.py (hunk 2 above)
Comment (idob-cloudify) [reply cloudify_cli/commands/nodes.py]: removed

**Comment by idob-cloudify** [comment]
fixed.

**Pull request merged**